show-changes

```diff
@@ -38,15 +39,28 @@ for section in security bugfixes changes updates; do
       exit 1
   esac
   echo
+  if [ "${section}" = security ]; then
+      FROM_KERNEL=$(git -C coreos-overlay show "${OLD}":sys-kernel/coreos-kernel/ | grep -m 1 'coreos-kernel-.*\.ebuild' | cut -d - -f 3 | cut -d . -f 1-3)
+      TO_KERNEL=$(git -C coreos-overlay show "${NEW}":sys-kernel/coreos-kernel/ | grep -m 1 'coreos-kernel-.*\.ebuild' | cut -d - -f 3 | cut -d . -f 1-3)
```

Comment:
Should this do the same submodule detection like the loop below?

Reply:
Yes :+1: